src/plugins/stepsform/stepsform.js

```diff
@@ -130,9 +133,59 @@ var componentName = "wb-steps",
 				$( form ).children( "input" ).hide();
 				wb.ready( $( elm ), componentName );
 			}
+
+			//Quiz code
+			if ( elm.classList.contains( "quiz" ) ) {
+
+				//Initialize all instances
+				let $instance = $( elm ),
+					numQuestion = $( ".steps-wrapper", $instance ).length; // Calculate number of questions
+
+				// Addition to UI (Ex: progress bar)
+				if ( !$.contains( $instance, "progress" ) ) {
+					$( "form", $instance ).prepend( "<label class='full-width'><span class='wb-inv'>" + i18nText.progresslabel + "</span><progress class='progressBar' max='" + numQuestion + "'></progress><p class='progressText' role='status'></p></label>" );
+				}
+
+				hideOtherSteps( evt );
+
+				$( document ).on( "click", quizSelector + " .steps-wrapper div.buttons > :button", hideOtherSteps );
+			}
 		}
 	},
 
+	/**
+	 * @method hideOtherSteps
+	 * @param {JavaScript element} e
+	 */
+	hideOtherSteps = function( e ) {
+
+		// Get wb-steps component
+		let steps,
+			currentElement = e.currentTarget;
+
+		if ( currentElement.classList.contains( "quiz" ) ) {
+			steps = currentElement;
+		} else {
+			steps = $( currentElement ).parentsUntil( quizSelector ).parent().get( 0 );
+		}
+
+		// Find the steps form context and validate it is a quiz
+		let currentTabId = $( "legend.wb-steps-active:first-child", steps ).parents().prevAll( ".steps-wrapper" ).length + 1,
```

Comment:
If the "if" above is not needed. Please combine these variables declarations with the ones above.